src/AdCredits.php

```diff
@@ -246,14 +246,13 @@ public static function process_available_discounts() {
 			return false;
 		}
 
-		$advertiser_id = Pinterest_For_Woocommerce()::get_setting( 'tracking_advertiser' );
+		$advertiser_id      = Pinterest_For_Woocommerce()::get_setting( 'tracking_advertiser' );
+		$discounts          = APIV5::get_ads_credit_discounts( $advertiser_id )['items'] ?? array();
```

Comment:
APIV5::get_ads_credit_discounts might throw a `PinterestApiException`.  can this handled?

Reply:
Good catch. It is handled downstream.

I have added `throws` to the dockblock for better visibility.